README.md

```diff
@@ -17,17 +17,17 @@ enabled by default in the GCE cluster.
 There are tons of node problems that could possibly affect the pods running on the
 node, such as:
 * Infrastructure daemon issues: ntp service down;
-* Hardware issues: Bad cpu, memory or disk;
+* Hardware issues: Bad CPU, memory, or disk;
```

Comment:
I think this is a bit obsessive, right?
For situation like this where both ways are correct [1], I'd prefer to just leave it as it. There is not much benefit of switching around.

[1] https://www.grammarly.com/blog/comma-before-and/

Reply:
Sometimes there's a certain amount of ambiguity which is fixed by using Oxford commas.

I'm not terribly attached to them. And given the damage above, if you're ok w/ a force push, I'm happy to wipe them out.